src/app/tests/suites/certification/Test_TC_ACE_1_6.yaml

```diff
@@ -11,662 +11,340 @@
-          [1685009266.594918][39716:39718] CHIP:DMG: ICR moving to [ResponseRe]
-          [1685009266.594926][39716:39718] CHIP:DMG: InvokeResponseMessage =
-          [1685009266.594928][39716:39718] CHIP:DMG: {
-          [1685009266.594930][39716:39718] CHIP:DMG: suppressResponse = false,
-          [1685009266.594932][39716:39718] CHIP:DMG: InvokeResponseIBs =
-          [1685009266.594935][39716:39718] CHIP:DMG: [
-          [1685009266.594937][39716:39718] CHIP:DMG: InvokeResponseIB =
-          [1685009266.594941][39716:39718] CHIP:DMG: {
-          [1685009266.594944][39716:39718] CHIP:DMG: CommandStatusIB =
-          [1685009266.594947][39716:39718] CHIP:DMG: {
-          [1685009266.594949][39716:39718] CHIP:DMG: CommandPathIB =
-          [1685009266.594951][39716:39718] CHIP:DMG: {
-          [1685009266.594954][39716:39718] CHIP:DMG: EndpointId = 0x0,
-          [1685009266.594956][39716:39718] CHIP:DMG: ClusterId = 0x4,
-          [1685009266.594959][39716:39718] CHIP:DMG: CommandId = 0x0,
-          [1685009266.594961][39716:39718] CHIP:DMG: },
-          [1685009266.594964][39716:39718] CHIP:DMG:
-          [1685009266.594966][39716:39718] CHIP:DMG: StatusIB =
-          [1685009266.594969][39716:39718] CHIP:DMG: {
-          [1685009266.594971][39716:39718] CHIP:DMG: status = 0x7e (UNSUPPORTED_ACCESS),
-          [1685009266.594973][39716:39718] CHIP:DMG: },
-          [1685009266.594976][39716:39718] CHIP:DMG:
-          [1685009266.594978][39716:39718] CHIP:DMG: },
-          [1685009266.594980][39716:39718] CHIP:DMG:
-          [1685009266.594982][39716:39718] CHIP:DMG: },
-          [1685009266.594985][39716:39718] CHIP:DMG:
-          [1685009266.594987][39716:39718] CHIP:DMG: ],
-          [1685009266.594990][39716:39718] CHIP:DMG:
-          [1685009266.594992][39716:39718] CHIP:DMG: InteractionModelRevision = 1
-          [1685009266.594994][39716:39718] CHIP:DMG: }
-      disabled: true
+      cluster: "Groups"
+      endpoint: Groups.Endpoint
+      command: "AddGroup"
+      arguments:
+          values:
+              - name: GroupID
+                value: 0x0104
+              - name: GroupName
+                value: ""
+      response:
+          error: UNSUPPORTED_ACCESS
 
     - label:
-          "TH sends a AddGroup Command to the Groups cluster with the GroupID
-          field set to 0x0104 and the GroupName set to an empty string. The
-          command is sent as a group command using GroupID 0x0103"
-      verification: |
-          ./chip-tool groups add-group 260 '' 0xffffffffffff0103 0
-      disabled: true
+          "Step 6: TH sends a AddGroup Command to the Groups cluster with the
+          GroupID field set to 0x0104 and the GroupName set to an empty string.
+          The command is sent as a group command using GroupID 0x0103"
+      cluster: "Groups"
+      endpoint: Groups.Endpoint
```

Comment:
Why is there an endpoint here?  I would not expect one for a group commmand.

Reply:
Updated.